src/zks_provider/mod.rs

```diff
@@ -1768,15 +1767,29 @@ mod tests {
             .with_chain_id(ERA_CHAIN_ID);
         let zk_wallet = ZKSWallet::new(wallet, None, Some(era_provider.clone()), None).unwrap();
 
-        let contract_address = zk_wallet
+        let contract: CompiledContract = serde_json::from_reader(
+            File::open("./src/abi/test_contracts/storage_combined.json").unwrap(),
```

Comment:
this will break if we run tests from another place that is not the root dir? Do we care?

Reply:
It may not be that important, but I fixed it anyway by following the same approach we used to search for `ContractDeployer.json` inside the `deploy` function. That should resolve the issue.